test/project_types/script/layers/application/extension_points_test.rb

```diff
@@ -68,6 +68,68 @@
     end
   end
 
+  describe ".all_languages" do
+    subject { Script::Layers::Application::ExtensionPoints.all_languages }
+
+    let(:scripts_beta_extension_points) { false }
+    let(:scripts_beta_languages) { false }
+
+    before do
+      ShopifyCLI::Feature
+        .expects(:enabled?)
+        .with(:scripts_beta_languages)
+        .returns(scripts_beta_languages)
+        .at_least_once
+      ShopifyCLI::Feature
+        .expects(:enabled?)
+        .with(:scripts_beta_extension_points)
+        .returns(scripts_beta_extension_points)
+        .at_least_once
```

Comment:
What's the rationale for setting up expectations instead of stubs here?

Reply:
Ah I was just copying the other tests, but you're right that they should be stubs. I changed all those occurrences for consistency